pkg/kubelet/cadvisor/cadvisor_linux_test.go

```diff
@@ -0,0 +1,66 @@
+// +build linux
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cadvisor
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	cadvisorfs "github.com/google/cadvisor/fs"
+	"k8s.io/kubernetes/pkg/kubelet/types"
+)
+
+func TestImageFsInfoLabel(t *testing.T) {
+	testcases := []struct {
+		description     string
+		runtime         string
+		runtimeEndpoint string
+		expectedLabel   string
+		expectedError   error
+	}{{
+		description:     "LabelDockerImages should be returned",
+		runtime:         types.DockerContainerRuntime,
+		runtimeEndpoint: "",
+		expectedLabel:   cadvisorfs.LabelDockerImages,
+		expectedError:   nil,
+	}, {
+		description:     "LabelCrioImages should be returned",
+		runtime:         types.RemoteContainerRuntime,
+		runtimeEndpoint: CrioSocket,
+		expectedLabel:   cadvisorfs.LabelCrioImages,
+		expectedError:   nil,
+	}, {
+		description:     "Cannot find valid imagefs label",
```

Comment:
Just glancing over since it's already lgtm-ed. Does it work for containerd? is it related to https://kubernetes.slack.com/archives/C0BP8PW9G/p1605899033213500?

Reply:
this captures existing behavior in code, so the test looks fine to me.